.changes/footer-with-experiments.md

```diff
@@ -1,9 +1,3 @@
-EXPERIMENTS:
-
-Experiments are only enabled in alpha releases of Terraform CLI. The following features are not yet available in stable releases.
```

Comment:
nit: Should the heading be left behind? We could have new featured launched under the experiments flag

Reply:
I think they'd add the header back in when we have the new features, we don't want the empty header showing up in CHANGELOGs.

Reply:
Oh. It is part of the changelog. my bad. Thanks!